specification/attestation/resource-manager/Microsoft.Attestation/preview/2018-09-01-preview/examples/Get_DefaultProvider.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "api-version": "2018-09-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "providers/Microsoft.Attestation/attestationProviders/sharedcus",
+        "name": "sharedcus",
+        "type": "Microsoft.Attestation/attestationProviders",
+        "location": "Central US",
```

Comment:
>Central US [](start = 21, length = 10)

Resources at tenant level are global due to ARM limitations. Do you need location here?

Reply:
Yes, because the RoutingType is ProxyOnly, Fanout. We still need location here to tell users which location this default provider is in.

Do you have any recommendation on whether these two Resources(	Microsoft.Attestation/defaultProviders and icrosoft.Attestation/locations/defaultProviders) should be tenant level or subscription level?
We set them to be tenant level because these resources are regional resources and not limit to customers subscription.